(node:3896706) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-session-42255',
  containerIp: '10.0.3.47',
  mitmPort: 42255,
  autoAccessFixed: 1
}
2024-11-08 10:11:50.793 - [Info] MITM Version: 2
2024-11-08 10:11:50.795 - [Info] Auto Access Enabled: true
2024-11-08 10:11:50.796 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-08 10:11:50.814 - [Info] SSH man-in-the-middle server for 10.0.3.47 listening on 10.0.3.1:42255
2024-11-08 10:11:51.775 - [Debug] [Connection] Attacker connected: 209.38.21.252 | Client Identification: SSH-2.0-Go
2024-11-08 10:11:53.224 - [Debug] [Auth] Attacker 209.38.21.252 trying to authenticate with 'none'
2024-11-08 10:11:53.225 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-08 10:11:53.666 - [Debug] [Auth] Attacker 209.38.21.252 trying to authenticate with 'password'
2024-11-08 10:11:53.667 - [Debug] [Auto Access] Attacker: 209.38.21.252, Threshold: 1, Attempts: 1
2024-11-08 10:11:53.667 - [Debug] [Auto Access] Compromising the honeypot
2024-11-08 10:11:53.667 - [Debug] [Auto Access] Adding the following credentials: 'default:1'
2024-11-08 10:11:53.925 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-08 10:11:53.927 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-08 10:11:53.927 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.47
2024-11-08 10:11:54.004 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-08 10:11:54.876 - [Debug] [EXEC] Noninteractive mode attacker command: uname -s -v -n -r -m
2024-11-08 10:11:55.415 - [Debug] [Connection] Attacker connected: 209.38.21.252 | Client Identification: SSH-2.0-Go
2024-11-08 10:11:55.793 - [Debug] [Connection] Attacker closed connection
2024-11-08 10:11:55.803 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-08 10:11:55.804 - [Debug] [LXC] Container's OpenSSH server closed connection
2024-11-08 10:11:56.966 - [Debug] [Auth] Attacker 209.38.21.252 trying to authenticate with 'none'
2024-11-08 10:11:56.966 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-08 10:11:57.396 - [Debug] [Auth] Attacker 209.38.21.252 trying to authenticate with 'password'
2024-11-08 10:11:57.397 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-08 10:11:57.397 - [Debug] [Auth] Attacker: 209.38.21.252 has so far made 1 attempts. Remaining: 5 attempts
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
